### ISSUE TITLE
How to train the models in smaller spochs

### ISSUE THREAD

**Issue by juliahane** [huggingface/transformers#9572]
Hi
I am under low compute hours, could you tell me how I can train finetune_trainer.py for smaller iterations and then continue retraining from the saved checkpoint to reproduce the same results as full training? what are the cares to be taken, and things to pay attention, thanks

**Comment by LysandreJik** [comment]
Hello, thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discusss.huggingface.co) instead?

Thanks!

**Comment by juliahane** [comment]
yes,. I asked but forum is inactive, please do not close it I really need
help on this.

**Comment by juliahane** [comment]
please do not close it I really need help on this and forrum is inactive

**Comment by thomwolf** [comment]
Our github issue policy is detailed in the related [ISSUE.md](https://github.com/huggingface/transformers/blob/master/ISSUES.md) and very clearly indicate we reserve the issue tracker for bugs and features request which this issue is not, as well as several other issues you have recently opened.

Feel free to post here a link to the thread you should open on the forum if you want to be visible in both location (though this should stay very exceptional).

The forum is NOT inactive, I see that you already have several answers to your [related post](https://discuss.huggingface.co/t/training-models-for-smaller-epochs-and-then-continue-trianing/3153). This is the place for discussion. NOT here in the issues.

Overall, please note that if you persist in not following the guidelines and open-source collaboration policies that we have defined and shared with the community on the repository in the [CODE_OF_CONDUCT](https://github.com/huggingface/transformers/blob/master/CODE_OF_CONDUCT.md), the [CONTRIBUTING](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md) and [ISSUES](https://github.com/huggingface/transformers/blob/master/ISSUES.md) documents, we reserve the right to take the moderation actions advised by GitHub in the [Community guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).

**Comment by juliahane** [comment]
Hi Thomas,
I did received some response, but those response were not helpful, this
needs a response from someone developed the codes, to know the small
details which can help,
please assist me with the issue, is there a way I could get a better help
in forume? so someone with more knowledge see the question?
thanks

**Comment by juliahane** [comment]
To me this question can safely be marked as a bug, since if one trains a
finetune_trainer.py till some epochs and start retraining from the saved
checkpoints, this does not get the same accuracy as full training, can I
file a bug for this issue? thanks

**Comment by thomwolf** [comment]
The maintainers have very clearly labelled it as NOT a bug ([here](https://github.com/huggingface/transformers/issues/9572#issuecomment-760020852)).

What you are asking here is a free consultation which unfortunately we don't provide at HuggingFace so here is the next step I think are the most adapted in the present case:
- you've probably reached the limit of the amount of consulting the community could provide for free, the best would be now to hire a consultant to build a solution for you
- regarding the current issue and your usage of the open-source repository, my mission is now to step up and protect the maintainers from the reprository to be able to conduct their mission of maintaining the repository and not diverging from their task to conduct free consulting missions. As such this is now the second warning I send you to follow the guidelines and open-source collaboration policies that we have defined and shared with the community on the repository in the CODE_OF_CONDUCT, the CONTRIBUTING and ISSUES documents (see my message [here](https://github.com/huggingface/transformers/issues/9572#issuecomment-760178500)).
- Which lead us to the last point: if I have to spend more time and send you a third warning to use the repository as it was designed for the community I will have to limit your ability to open issues on our repositories following the GitHub Community guidelines as I explained to you in my message [here](https://github.com/huggingface/transformers/issues/9572#issuecomment-760178500). That would be the first time I have to do that with a community of thousand of people so please just use our forum tools like we designed them according to our guidelines stated [here](https://github.com/huggingface/transformers/issues/9572#issuecomment-760178500).
